ui/src/notebooks/context/notebook.tsx

```diff
@@ -14,23 +14,27 @@ export interface PipeMeta {
 // TODO: this is screaming for normalization. figure out frontend uuids for cells
```

Comment:
I think we can delete this whole file? maybe later, but i got confused as to how any of this worked